src/pasta/curves.rs

```diff
@@ -700,5 +702,5 @@ macro_rules! new_curve_impl {
     };
 }
 
-new_curve_impl!(Ep, EpAffine, Fp, Fq);
-new_curve_impl!(Eq, EqAffine, Fq, Fp);
+new_curve_impl!(Ep, EpAffine, Fp, Fq, b"halo2_____pallas");
+new_curve_impl!(Eq, EqAffine, Fq, Fp, b"halo2______vesta");
```

Comment:
These _definitely_ will need changing to be more descriptive as to their purpose 🙂

Reply:
This will change again when we switch to simplified SWU for hash-to-curve.